tests/codeception/backend/acceptance/LoginCept.php

```diff
@@ -20,7 +20,6 @@
 $I->see('Password cannot be blank.', '.help-block');
 
 $I->amGoingTo('try to login with wrong credentials');
-$I->expectTo('see validations errors');
```

Comment:
@klimov-paul the real expectTo() is already exist in line 27, so this one in line 23 is not needed.